packages/fury/README.md

```diff
@@ -155,28 +155,28 @@ export function detect(source[, method]) {
   return source.match(/some-test/i) !== null;
 }
 
-export function parse({minim, generateSourceMap, mediaType, source}, done) {
+export async function parse({minim, generateSourceMap, mediaType, source}) {
```

Comment:
This comment may be irrelevant, as I don't know if `async` is used anywhere else, but in case this is the only usage, I'd recommend to use `Promise` instead. `async/await` appends generators to support this feature in the compiled code, while in this case it's only in order for `parse`, `validate` (and others) to return a Promise.

Returning a plain Promise may reduce the package's size.

Reply:
We're using Promises everywhere because we still support Node 6, however the documentation here uses async for brevity/simplicity. I think developers are smart enough to use Promises if they use older Node.

I don't see how it would change the package size, the Promise syntax if anything will be a little longer and thus make the README file sizes larger?